styleguide/components/31-molecules/nav-filter/nav-filter.twig

```diff
@@ -0,0 +1,7 @@
+<nav class="filter--nav">
+  {% include '@list' with {
```

Comment:
Missing hidden h2 and aria-labelledby to name this nav region.
See breadcrumbs, paging...